CHANGELOG.md

```diff
@@ -2,7 +2,7 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
-## [Unreleased][unreleased]
+## [Unreleased]
```

Comment:
The lower case version makes a link. See the last line, you need to make the first letter capital there.